nupic/embodied/envs/real_robots/wrappers.py

```diff
@@ -36,42 +35,43 @@ class WrapRobot(object):
 
     crop_obs: Specifies whether the retina image should be cropped from
              240x320x3 to 180x180x3 (mainly cuts off white space)
-    '''
-    def __init__(self, env,crop_obs=False):
+    """
+
+    def __init__(self, env, crop_obs=False):
         self._env = env
         self.crop_obs = crop_obs
 
-        self._env.action_space = self._env.action_space['joint_command']
+        self._env.action_space = self._env.action_space["joint_command"]
         if self.crop_obs:
-            self._env.observation_space = spaces.Box(low=0, high=255.0,shape=(180,180,3), dtype='float32')
```

Comment:
Breaks down a line longer than 88. This was autoformatted with black, on the modified code you can see the style black applies.